src/codegen/trampoline_from_glib.rs

```diff
@@ -70,10 +70,9 @@ pub fn from_glib_xxx(transfer: library::Transfer, is_borrow: bool) -> (String, S
 fn is_need_type_name(env: &Env, type_id: library::TypeId) -> bool {
     if type_id.ns_id == library::INTERNAL_NAMESPACE {
         use crate::library::{Fundamental::*, Type::*};
-        match *env.type_(type_id) {
-            Fundamental(fund) if fund == Utf8 => true,
-            Fundamental(fund) if fund == Filename => true,
-            Fundamental(fund) if fund == OsString => true,
+        match env.type_(type_id) {
+            // Fundamental(Utf8 | Filename | OsString) => true,
+            Fundamental(Utf8) | Fundamental(Filename) | Fundamental(OsString) => true,
```

Comment:
Nested `|` (or) patterns will come in Rust 2021!!

https://blog.rust-lang.org/2021/05/11/edition-2021.html#or-patterns-in-macro_rules